test/Subst/TypeError.py

```diff
@@ -85,8 +85,36 @@ def func(a, b, c):
 
 test.run(status=2, stderr=expect)
 
+# callable exceptions:
```

Comment:
You should be able to to use test_main.c from test/fixtures instead of adding another file write in the test.

Reply:
done: deacd08